doc/source/dev/contributor/continuous_integration.rst

```diff
@@ -0,0 +1,90 @@
+.. _continuous-integration:
+
+======================
+Continuous Integration
+======================
+
+Continuous integration is part of our development process and ensure that
+every piece of code or documentation which is contributed to SciPy is working
+and does not have unforeseen effects.
+
+.. note:: Before submitting or updating your PR, please ensure that you tested
+          locally your changes. See :ref:`pr-checklist` and :ref:`runtests`.
+
+Workflows
+=========
+
+We run more than 20 different workflows with different version of the
```

Comment:
`version` -> `versions`

Reply:
Thanks Tyler for your review! I will check all that. BTW, will you be able to join the meetup?